v2/README.md

```diff
@@ -560,7 +565,7 @@ func ExampleExecuteOperation() {
 		if err != nil {
 			panic(err)
 		}
-		fmt.Println(out.String()) // Output: {"data":{"hello":"world"}}
+		fmt.Println(out) // Output: {"data":{"hello":"world"}}
```

Comment:
I am still getting an error instead of the suggested output here:
```json
{"errors":[{"message":"String cannot represent non-string value: \"\"","path":[]}],"data":null}
```
I'll need more time to trace why this is happening, so in case the cause is obvious to someone, feel free to point me in the right direction.